README.md

```diff
@@ -23,3 +23,21 @@ inefficient.
 
 What is the big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+
+The first for loop that initializes the nodes to infinity runs(except source node)for over all the nodes(vertices) so this is (v). The second for loop also iterates over all the nodes again to make sure they have all been visited which is (v). The third for loop which finds the unvisited node with the smallest distance, for each iteration in the second loop this runs v times so this is v*v or v^2. The fourth for loop runs for the amount of edges total, which is the total neighbors which is (e). This comes out to being V + V + V^2 + E = $\Theta$(V^2 +E) as we ignore the other 2 V's being that they are smaller than V^2. 
```

Comment:
Can you bound $E$ in some way? In particular, does the number of edges really matter asymptotically?